src/compiler/checker.ts

```diff
@@ -1776,6 +1777,20 @@ namespace ts {
             }
         }
 
+        function tryGetMemberInModuleExportsAndProperties(memberName: __String, moduleSymbol: Symbol): Symbol | undefined {
+            const symbolTable = getExportsOfModule(moduleSymbol);
+            if (symbolTable) {
+                let symbol = symbolTable.get(memberName);
+                if (!symbol) {
+                    const exportEquals = resolveExternalModuleSymbol(moduleSymbol);

```

Comment:
It also seems like the first attempt (i.e. the exports) could be attempted by calling `tryGetMemberInModuleExports`.